config/locales/en.yml

```diff
@@ -501,7 +501,8 @@ en:
         wrc_feedback_requested: ""
         wrc_incidents: "List of numbers referencing any incidents that you would like the WRC's feedback on."
         wic_feedback_requested: ""
-        wic_incidents: "List of numbers referencing any incidents that you would like the WIC's feedback on."
+        wic_incidents: "List of numbers referencing any incidents that you would like the WIC's feedback on. <strong>%{wic_incident_alert}</strong>"
```

Comment:
I am still pretty sure you need to call this `wic_incidents_html` (both here and in the code where you call `I18n.t('...')`). Otherwise the HTML renderer will just print the literal `<strong>` string as readable text to your screen, without treating it as parseable HTML

Reply:
Hm :thinking: I wonder why it works though, normally the `_html` convention is "baked into" the Rails I18N engine...

Reply:
Ah, it's because you're manually doing `.html_safe` above. Calling the key `foo_bar_html` would do that automagically for you :wink: